tools/gsc_high_dimensions_visualisation/high_dim_visu.R

```diff
@@ -272,19 +269,24 @@ if (opt$visu_choice == 'tSNE') {
     ggplot(embedding, aes(x=V1, y=V2)) +
       geom_point(size=1, color='deepskyblue4') +
       gg_legend +
-      xlab("") +
-      ylab("") +
+      xlab("t-SNE 1") +
```

Comment:
honestly, the added value of this legend is limited. Would it be possible to add useful information ?

Reply:
Rtsne don't give any additional information about dimensions like PCA. I always give axis labels this is why I changed to t-SNE 1 and t-SNE 2 (or Dim 1/Dim 2). If this is not relevant enough, I can get back to empty labels.